cg/constants/delivery.py

```diff
@@ -153,20 +153,21 @@
     {"fusion", "arriba"},
     {"fusion", "star-fusion"},
     {"fusion", "fusioncatcher"},
-    {"cram"},
     {"fusioncatcher-summary"},
     {"fusioninspector"},
     {"fusionreport", "research"},
     {"fusioninspector-html", "research"},
     {"arriba-visualisation", "research"},
     {"multiqc-html", "rna"},
-    {"software-versions"},
-    {"qc-metrics"},
-    {"multiqc-json"},
     {"delivery-report"},
+    {"vcf-fusion"},
+    {"gene-counts"},
 ]
 
-RNAFUSION_ANALYSIS_SAMPLE_TAGS: list[set[str]] = []
+RNAFUSION_ANALYSIS_SAMPLE_TAGS: list[set[str]] = [
+    {"cram"},
+    {"cram-index"},
```

Comment:
I have seen these tags also in MIP. Maybe we can start a
`class AnalysisSampleTags(StrEnum):`
?

Reply:
This is a good point. Let's do it in a different PR. Opened issue here https://github.com/Clinical-Genomics/cg/issues/2743